examples/integration/xcodeproj_targets.bzl

```diff
@@ -6,6 +6,8 @@ load(
 )
 
 XCODEPROJ_TARGETS = [
+    "//examples/integration/CommandLine/CommandLineTool",
+    "//examples/integration/CommandLine/Tests:CommandLineToolLibSwiftTests",
```

Comment:
Can you set `target_environments = ["device"],` for both of these? Will be needed long with my other suggestion.